include/motor-control/core/stepper_motor/motor_encoder_background_timer.hpp

```diff
@@ -32,6 +32,8 @@ class BackgroundTimer {
         if (!_interrupt_handler.has_active_move()) {
             // Refresh the overflow counter if nothing else is doing it
             std::ignore = _motor_hardware.get_encoder_pulses();
```

Comment:
Because `stall_detected` already checks the encoder pulses, we can just get rid of this line

Reply:
So true!